deploy/modules/gcp/tidb-operator/main.tf

```diff
@@ -0,0 +1,156 @@
+resource "google_container_cluster" "cluster" {
+  name       = var.gke_name
+  network    = var.vpc_name
+  subnetwork = var.subnetwork_name
+  location   = var.gcp_region
+  project    = var.gcp_project
+
+  master_auth {
+    username = ""
+    password = ""
+
+    // due to https://github.com/terraform-providers/terraform-provider-google/issues/3369
+    client_certificate_config {
+      issue_client_certificate = false
+    }
+  }
+
+  master_authorized_networks_config {
+    cidr_blocks {
+      cidr_block = "0.0.0.0/0"
+    }
+  }
+
+  ip_allocation_policy {
+    use_ip_aliases = true
+  }
+
+  // See https://kubernetes.io/docs/setup/best-practices/cluster-large/#size-of-master-and-master-components for why initial_node_count is 5. The master node should accommodate up to 100 nodes like this
+  remove_default_node_pool = true
+  initial_node_count       = 5
+
+  min_master_version = var.gke_version
+
+  lifecycle {
+    ignore_changes = [master_auth] // see above linked issue
+  }
+
+  maintenance_policy {
+    daily_maintenance_window {
+      start_time = "01:00"
```

Comment:
Then it would be better if users can customize the time, because 01:00 GMT is 9:00 Asia/Beijing

Reply:
I don't see how this get resolved